pyscf/dft/r_numint.py

```diff
@@ -191,7 +217,7 @@ def r_vxc(ni, mol, grids, xc_code, dms, spin=0, relativity=0, hermi=1,
                 matLL[idm] += _vxc2x2_to_mat(mol, ao[:2], weight, rho, vrho,
                                              mask, shls_slice, ao_loc)
                 if with_s:
-                    matSS[idm] += _vxc2x2_to_mat(mol, ao[2:], weight, rho, vrho,
+                    matSS[idm] += _vxc2x2_to_mat_S(mol, ao[2:], weight, rho, vrho,
```

Comment:
Does it equal to `matSS[idm] - _vxc2x2_to_mat(mol, ao[2:], ...`?

Reply:
I think it does not equal because the density-dependent part should not change. The only difference is there should be a minus of the small component for the spin density-dependent part.